.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "packages/contracts/lib/forge-std"]
+	path = packages/contracts/lib/forge-std
+	url = https://github.com/foundry-rs/forge-std
```

Comment:
We now install forge-std as a git submodule as recommended by foundry. When the user installs us in their project using `forge install`, our dependency on forge-std will also be installed and our contracts will have everything they need to compile properly.